invenio_pidstore/models.py

```diff
@@ -55,12 +55,12 @@
 from datetime import datetime
 
 from sqlalchemy.exc import SQLAlchemyError
+from invenio_db import db, InvenioDB
 
-from invenio_base.globals import cfg
-from invenio_ext.sqlalchemy import db
 from invenio_utils.text import to_unicode
```

Comment:
We can't use invenio utils. Perhaps you can use another library or simply copy the code into the library.